misc/FFNx.cfg

```diff
@@ -15,46 +15,73 @@
 # COMMON OPTIONS FOR BOTH FF7 AND FF8
 ###############################################################################
 
-## RENDERING
+#########################
+## DISPLAY
+#########################
 
+#[RENDERING BACKEND]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Available choices are:
-### SUGGESTED:
 # - OpenGL ( default )
 # - Direct3D11 ( valid alternative for OpenGL)
-### NOT SUGGESTED (TECHNICAL POC):
-# - Vulkan ( slow performance)
-# - Direct3D12 ( slow performance )
+# - Vulkan ( Tech Preview: Not Recommened )
+# - Direct3D12 ( Tech Preview: Not Recommened )
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #renderer_backend = OpenGL
 
-# Enable fullscreen mode.
+#[FULLSCREEN]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # If off, it will run in window mode.
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #fullscreen = no
 
+#[RESOLUTION]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Resolution of the game.
-# If any of the two is 0, the driver will use original game resolution ( 640x480 ) in window mode.
-# Your desktop resolution in fullscreen mode otherwise.
+# Default (value = 0):
+# - Window mode will use 640x480
+# - Fullscreen mode will use your desktop resolution
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #window_size_x = 0
 #window_size_y = 0
 
+#[INTERNAL RESOLUTION SCALE]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Internal Scale used it Perserve Aspect is not enabled.

```

Comment:
Needs a rephrase or something is missing?

Reply:
Can you describe what this setting is doing for me ?